pkg/db/db_test.go

```diff
@@ -4093,6 +4093,366 @@ func TestReadAllManifestsAllReleases(t *testing.T) {
 	}
 }
 
+func TestDBWriteReadUnsynced(t *testing.T) {
+	tcs := []struct {
+		Name               string
+		SyncDataToWrite    []GitSyncData
+		ExpectedSyncEvents map[int][]EnvApp
+	}{
+		{
+			Name: "Read Unsynced apps for transformer",
+			SyncDataToWrite: []GitSyncData{
+				{
+					AppName:       "app-1",
+					EnvName:       "env-1",
+					TransformerID: EslVersion(1),
+					SyncStatus:    UNSYNCED,
+				},
+				{
+					AppName:       "app-2",
+					EnvName:       "env-1",
+					TransformerID: EslVersion(2),
+					SyncStatus:    UNSYNCED,
+				},
+				{
+					AppName:       "app-3",
+					EnvName:       "env-3",
+					TransformerID: EslVersion(3),
+					SyncStatus:    UNSYNCED,
+				},
+				{
```

Comment:
could we add a 2nd case here?
Could be exactly the same, but one addition appEnv that is already synced.